### PR TITLE
Sztab-WOSP-Bialystok/wosp-staff-issues#7

### PR DESCRIPTION
**Pull request by TheUnDc**

### CODE REVIEW

File: app/concerts/page.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+"use client";
+
+import { Container, Row, Col, Card } from "react-bootstrap";
+
+export default function Page() {
+	return (
+		<Row>
```
Comment (Freesebee) [inline app/concerts/page.tsx:7]:
```suggestion
```

File: app/concerts/page.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+								adipisicing elit. Commodi debitis quas aperiam
+								rem earum aliquid atque nobis enim dignissimos
+								vero sunt nesciunt vel, quod ex molestias totam
+								iusto possimus impedit. 
+							</Card.Body>
+						</Card>
+					</Col>
+				</Row>
+			</Container>
+		</Row>
```
Comment (Freesebee) [inline app/concerts/page.tsx:31]:
```suggestion
```

File: app/marathons/page.tsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+import { EXTERNAL_SOURCE_KEYS } from "../consts/external-source-keys.const";
+
+export default function Page() {
+	const data = externalSource;
+	const marathonExternal = data.find(
+		(external) => external.key == EXTERNAL_SOURCE_KEYS.MARATHON_FORM
+	);
+	return (
+		<Container className="m-2 mt-sm-3 m-xl-5">
+			<Col xl={{ span: 8, offset: 2 }} className="my-3 img-overlay">
```
Comment (Freesebee) [inline app/marathons/page.tsx:14]:
```suggestion
                        <Row>
			                  <Col xl={{ span: 8, offset: 2 }} className="my-3 img-overlay">
```

File: app/marathons/page.tsx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+							elit. Totam optio ullam sit cum rerum, aut ex eos
+							natus alias eveniet pariatur magni neque perferendis
+							aperiam non voluptatum officia reprehenderit
+							reiciendis.
+						</Accordion.Body>
+					</Accordion.Item>
+				</Accordion>
+			</Col>
+
+			<Row>
```
Comment (Freesebee) [inline app/marathons/page.tsx:61]:
```suggestion
```

**Review by Freesebee — CHANGES_REQUESTED** [review]
❤️ but I will add placeholder data, and consult layout with coordinator

File: app/components/navbar/navigation.ts [hunk 5]
```diff
@@ -15,6 +15,8 @@ export enum Links {
   VOLOUNTEERS = "/volounteers",
   MONEY_BOX = "/money-box",
   GALLERY = "https://www.facebook.com/bialystokwosp/photos",
+  CONCERTS = "/concerts",
```
Comment (Freesebee) [inline app/components/navbar/navigation.ts:18]:
```suggestion
```

File: app/marathons/page.tsx [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+	const marathonExternal = externalSourceData.find(
+		(external) => external.key == EXTERNAL_SOURCE_KEYS.MARATHON_FORM
+	);
+	return (
+		<Container className="my-3 my-xl-5">
+			<Row>
+				<Col xl={{ span: 8, offset: 2 }} className="my-2 img-overlay">
+					<h1 className="text-center">Zapisy na bieg trwają!</h1>
+					<p className="text-center px-5 px-md-0">
+						Biegnij dla zdrowia i zabawy! Dołącz do tysięcy biegaczy
```
Comment (Freesebee) [inline app/marathons/page.tsx:25]: When making responsive UI design it's important to remember polish grammar rules. [As a general rule, you should not leave single-letter conjunctions and prepositions at the end of a line](https://sjp.pwn.pl/poradnia/haslo/Sieroty-na-koncu-wiersza;18684.html#:~:text=W%20wierszach%20tekstu%2C%20kt%C3%B3re%20licz%C4%85%20powy%C5%BCej%2040%20znak%C3%B3w%2C,dopuszczalne%20tylko%20przy%20bardzo%20w%C4%85skim%20%C5%82amie%20%28np.%20gazetowym%29.)

You can achievie this by using non-breaking space `&nbsp;`

```suggestion
						Biegnij dla zdrowia i&nbsp;zabawy! Dołącz do tysięcy biegaczy
```

File: app/marathons/page.tsx [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+	Button,
+	Card,
+	Accordion,
+	ListGroup,
+} from "react-bootstrap";
+import externalSourceData from "../../public/info/external-sources.json";
+import { EXTERNAL_SOURCE_KEYS } from "../consts/external-source-keys.const";
+
+export default function Page() {
+	const marathonExternal = externalSourceData.find(
```
Comment (Freesebee) [inline app/marathons/page.tsx:16]: As I learned from @Kompanel 😄 - use states to reduce calling `find()` method
```suggestion
	const [marathonExternal ] = useState<ExteralSource>(externalSourceData.find((x) => x.key ==EXTERNAL_SOURCE_KEYS.BIALYSTOK_WEB)
```

**Remember to import useState and other types like in [Honorary Patronage page](https://github.com/Sztab-WOSP-Bialystok/wosp-staff/tree/main/app/home/sections/honorary-patriage)**

File: app/marathons/page.tsx [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+							</Accordion.Header>
+
+							<Accordion.Body>
+								<ListGroup as="ol" numbered>
+									<ListGroup.Item variant="light">
+										Należy otworzyć formularz zgłoszeniowy
+										dostępny poniżej.
+									</ListGroup.Item>
+									<ListGroup.Item variant="dark">
+										Wybrać odpowiednią opcję w zależności od
```
Comment (Freesebee) [inline app/marathons/page.tsx:44]:
```suggestion
										Wybrać odpowiednią opcję w&nbsp;zależności od
```

File: app/marathons/page.tsx [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+										Uzupełnić dane.
+									</ListGroup.Item>
+									<ListGroup.Item variant="dark">
+										Kliknąć "prześlij formularz".
+									</ListGroup.Item>
+									<ListGroup.Item variant="light">
+										Oczekiwać na potwierdzenie z naszej
+										strony 🙂 ( UWAGA: wiadomości zwrotne
+										NIE SĄ generowane automatycznie,
+										sprawdzamy formularze i odpisujemy
```
Comment (Freesebee) [inline app/marathons/page.tsx:58]:
```suggestion
										sprawdzamy formularze i&nbsp;odpisujemy
```

File: app/marathons/page.tsx [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+
+				<Col
+					xl={{ span: 8, offset: 2 }}
+					style={{ marginTop: "8rem", marginBottom: "1rem" }}
+				>
+					<Card>
+						<Card.Body>
+							<div className="text-center text-md-start">
+								<h1 className="info">
+									Nie czekaj, zarejestruj się i biegnij z
```
Comment (Freesebee) [inline app/marathons/page.tsx:120]:
```suggestion
									Nie czekaj, zarejestruj się i&nbsp;biegnij z&nbsp;
```

File: app/marathons/page.tsx [hunk 11]
```diff
@@ -0,0 +1,140 @@
...
+									nami!
+								</h1>
+								<h6>
+									Wejdź w formularz i wypełnij (UWAGA:
```
Comment (Freesebee) [inline app/marathons/page.tsx:124]:
```suggestion
									Wejdź w formularz i&nbsp;wypełnij (UWAGA:
```

File: app/marathons/page.tsx
Comment (Freesebee) [inline app/marathons/page.tsx:1]: From upper staff: 
- remove all accordion, listGroup components: image
- remo space between cards: image
- remove "Uranium-235" styling from second card to match cards in other pages: image

**Review by Freesebee — CHANGES_REQUESTED** [review]
🤗 🎖️

**Review by Kompanel — APPROVED** [review]

File: app/marathons/page.tsx
Comment (TheUnDc) [reply app/marathons/page.tsx:1]: done

**Review by Freesebee — APPROVED** [review]
❤️

**Pull request merged**